vimeo-networking/src/main/java/com/vimeo/networking/logging/ClientLogger.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Vimeo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.vimeo.networking.logging;
+
+import com.vimeo.networking.VimeoClient;
+
+/**
+ * Simple utility class to delegate logging when it is not known whether VimeoClient has been initialized yet
+ * <p/>
+ * Created by zetterstromk on 6/15/16.
+ */
+public final class ClientLogger {
```

Comment:
Two more suggestions from our conversation:
1) VimeoClient's access to the debug logger on the configuration gets more closed off and everything routes through client logger (which also implements the interface to give it a tighter contract). This way this class is the defacto source for logging in vimeo client.

2) DebugLoggingInterface becomes an abstract class with method bodies that just log to system.out (giving it default functionality). And then another class could extend it - this class could have static accessors and maybe be initialized with VimeoClient's debug logging functionality and if that doesnt' exist, the class will fallback to it's superclasses default functionality. Pros - it removes the need for a class. Cons - the contract is a little looser.